docs/marko-testing-library/setup.md

```diff
@@ -0,0 +1,105 @@
+---
+id: setup
+title: Setup
+sidebar_label: Setup
+---
+
+`Marko Testing Library` is not dependent on any test runner, however it is
+dependent on the test environment. These utilities work for testing both server
```

Comment:
```suggestion
dependent on browser globals. These utilities work for testing both server

Reply:
What are "these utilities" in this paragraph?

Reply:
Maybe `test environment` is not the most clear here. Basically I am trying to express that if your tooling does not pick up on the `browser` field in the package.json then your tests will be running in the SSR mode. For jest this just means enabling the `browser` option. Just having a global JSDOM for example (as is the default for jest) will not work by itself. It's a bit of a strange thing to explain to new comers though, do you have some thoughts on how to better describe this?

`these utilities` really just mean the api's exposed by the package. Maybe `these apis` would be better?

Reply:
Does that mean you can only test built files and not individual components? AFAIK most tests that require from the src folder aren't really aware of the packaging system.

Reply:
You can test individual components, the key is that by default the server side version of the component is resolved. For a client side version of a component to be resolved, the bundler, or test runner, needs to be aware of the [browser field](https://github.com/defunctzombie/package-browser-field-spec).

In Marko components are compiled to an optimized SSR streaming API or a VDOM api. The browser field is used to provide an appropriate runtime based on the environment.